test/basic.js

```diff
@@ -1,76 +1,73 @@
-'use strict';
+import assert from "power-assert";
+import * as ShioriJK from "shiorijk";
```

Comment:
__[jshint]__
*  'import' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->